### PR TITLE
ci: init flake8 linting

### PR DESCRIPTION
**Pull request by simonseyock**
This is based on https://github.com/mapproxy/mapproxy/pull/861

### CODE REVIEW

File: mapproxy/cache/file.py [hunk 1]
```diff
@@ -45,14 +47,15 @@ def __init__(self, cache_dir, file_ext, directory_layout='tc',
...
 
     def tile_location(self, tile, create_dir=False, dimensions=None):
-        if dimensions is not None and len(dimensions)>0:
+        if dimensions is not None and len(dimensions) > 0:
             items = list(dimensions.keys())
             items.sort()
             dimensions_str = ['{key}-{value}'.format(key=i, value=dimensions[i].replace('/', '_')) for i in items]
-            cache_dir = os.path.join(self.cache_dir, '_'.join(dimensions_str))
+            # todo: cache_dir is not used. should it get returned or removed?
+            cache_dir = os.path.join(self.cache_dir, '_'.join(dimensions_str))  # noqa
```
Comment (simonseyock) [inline mapproxy/cache/file.py:58]: This might be an error that existed before. This could get adressed in a future PR

Comment (simonseyock) [reply mapproxy/cache/file.py:58]: There also already exists an issue for

File: mapproxy/cache/tile.py [hunk 2]
```diff
@@ -122,13 +123,10 @@ def load_tile_coord(self, tile_coord, dimensions=None, with_metadata=False):
             [tile_coord], dimensions=dimensions, with_metadata=with_metadata,
         )[0]
 
-
     def load_tile_coords(self, tile_coords, dimensions=None, with_metadata=False):
         tiles = TileCollection(tile_coords)
         rescale_till_zoom = 0
         if self.rescale_tiles:
-            rescaled_tiles = {}
```
Comment (weskamm) [inline mapproxy/cache/tile.py:130]: removal necessary?

File: mapproxy/test/system/test_seed.py [hunk 3]
```diff
@@ -236,46 +240,46 @@ def test_seed_mbtile(self):
...
-            ImageSource(tile_image_buf,
-                image_opts=ImageOptions(format='image/png')))
+                    ImageSource(tile_image_buf,
+                                image_opts=ImageOptions(format='image/png')))
 
     def test_reseed_mbtiles(self):
-        seed_conf  = load_seed_tasks_conf(self.seed_conf_file, self.mapproxy_conf)
-        tasks, cleanup_tasks = seed_conf.seeds(['mbtile_cache']), seed_conf.cleanups(['cleanup_mbtile_cache'])
+        seed_conf = load_seed_tasks_conf(self.seed_conf_file, self.mapproxy_conf)
+        tasks, _ = seed_conf.seeds(['mbtile_cache']), seed_conf.cleanups(['cleanup_mbtile_cache'])
```
Comment (weskamm) [inline mapproxy/test/system/test_seed.py:259]: interesting variable rename?

**Review by weskamm — APPROVED** [review]
Hope i did not miss something serious, but have only 2 minor remarks

File: mapproxy/cache/tile.py (hunk 2 above)
Comment (simonseyock) [reply mapproxy/cache/tile.py:130]: it's an unused variable

File: mapproxy/test/system/test_seed.py (hunk 3 above)
Comment (simonseyock) [reply mapproxy/test/system/test_seed.py:259]: the variable is not used. if it is named `_` it is ignored by the linter.

Comment (simonseyock) [reply mapproxy/test/system/test_seed.py:259]:
```suggestion
        tasks = seed_conf.seeds(['mbtile_cache'])
        seed_conf.cleanups(['cleanup_mbtile_cache'])
```

**Pull request merged**